### PR TITLE
Update base image and default to postgres 13.

### PR DESCRIPTION
**Pull request by fitzy101**
This updates the base image to Postgres 13, as well as including a new template for postgres 13.

### CODE REVIEW

File: templates/postgres.template.yml [hunk 1]
```diff
@@ -167,36 +168,36 @@ run:
...
+      filename: "/etc/postgresql/13/main/pg_hba.conf"
       from: /^#local +replication +postgres +peer$/
       to: "local replication postgres  peer"
 
   # allow all to connect in with md5 auth
   - replace:
-      filename: "/etc/postgresql/12/main/pg_hba.conf"
-      from: /^host.*all.*all.*127.*$/
+      filename: "/etc/postgresql/13/main/pg_hba.conf"
+      from: /^host.*all.*all.*137.*$/
```
Comment (CvX) [inline templates/postgres.template.yml:193]: Search-and-replace-all strikes again! 😄

I believe it should remain unchanged?

```suggestion
      from: /^host.*all.*all.*127.*$/
```

Comment (fitzy101) [reply templates/postgres.template.yml:193]: I thought I got all these! Good spotting.

**Comment by xfalcox** [comment]
Looking good!

**Comment by fitzy101** [comment]
@xfalcox I've pushed a few commits since Friday, and reverted the change to the image as that will need to change once this lands on master and the new image is built and tagged. I'll follow up with another commit to update that once the image is built. Would you mind reviewing again? I've also tested in a variety of disk usage scenarios, and have a Meta post ready to go once things are merged.

**Pull request merged**